src/routes/blog/post/case-study-langx/+page.markdoc

```diff
@@ -60,4 +60,4 @@ With that said, the team has a number of features in the works, which include:
 - A [Language Copilot](https://docs.langx.io/litepaper/library/language-copilot) to provide personalized feedback and language suggestions when interacting with other users via AI *(this will be a paid feature)*
 - A new [LangX token](https://docs.langx.io/litepaper/token/token) to create a structured reward system for users, incentivizing learning on the platform
 
-Behic and LangX have had an inspiring journey, and we appreciate how they have leveraged Appwrite to empower language learners around the globe. Learn more about their work and team by visiting their [website](https://langx.io/) and [documentation](https://docs.langx.io/litepaper).
\ No newline at end of file
+Xue and LangX have had an inspiring journey (he has grown to become a polyglot, too), and we appreciate how they have leveraged Appwrite to empower language learners around the globe. Learn more about their work and team by visiting their [website](https://get.langx.io/) and [documentation](https://docs.langx.io/litepaper). You can also contribute to the development of LangX by visiting their [GitHub organization](https://github.com/langx) and [Discord community](https://discord.langx.io).
```

Comment:
https://get.langx.io/ goes to their app and shows:

![image](https://github.com/appwrite/website/assets/1477010/53cd534f-5d53-410e-9018-c6ae03f52372)

which can be confusing for someone who doesn't know much about LangX. I think their [info site](https://langx.io/) would be much better

Reply:
Updating